manifests/n/Neptop/VisionTeacher/1.7.6.0/Neptop.VisionTeacher.locale.en-US.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Neptop.VisionTeacher
+PackageVersion: 1.7.6.0
+PackageLocale: en-US
+Publisher: Netop Solutions AS

```

Comment:
```suggestion
Publisher: Impero Solutions Ltd
```

Reply:
impero bought Netop in January 2021. Should this be updated? 
https://www.imperosoftware.com/press-release/impero-software-acquires-netop-to-expand-its-global-education-and-corporate-product-offerings/